src/main/kotlin/pt/up/fe/ni/website/backend/service/activity/AbstractActivityService.kt

```diff
@@ -1,16 +1,18 @@
-package pt.up.fe.ni.website.backend.service
+package pt.up.fe.ni.website.backend.service.activity
 
 import org.springframework.data.repository.findByIdOrNull
 import org.springframework.stereotype.Service
 import pt.up.fe.ni.website.backend.model.Activity
 import pt.up.fe.ni.website.backend.repository.ActivityRepository
+import pt.up.fe.ni.website.backend.service.AccountService
+import pt.up.fe.ni.website.backend.service.ErrorMessages
 
 @Service
-abstract class ActivityService<T : Activity>(
-    private val repository: ActivityRepository<T>,
-    private val accountService: AccountService
+abstract class AbstractActivityService<T : Activity>(
```

Comment:
I'm unsure if this is the best way to fix the problem... I needed a way to instantiate an ActivityService. I'd appreciate suggestions for a better pattern